x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtRealmTests.java

```diff
@@ -108,6 +104,122 @@ public void shutdown() throws Exception {
         terminate(this.threadPool);
     }
 
+    /**
+     * Generate a static JWT realm config, and request headers, which the realm can successfully authenticate.
+     *
+     * Print these artifacts, so they can be imported into this integration test project for verifying loading of JWT realm config:
+     *  - x-pack/plugin/security/qa/smoke-test-all-realms/build.gradle
+     *
+     * If something needs tweaking, change the code here and re-run it. The new artifacts can be used to update that smoke test project.
+     * @throws Exception Error if something goes wrong.
+     */
+    public void testCreateJwtSmokeTestRealm() throws Exception {
```

Comment:
The output of this test is the static JWT realm config, client secret, and signed JWT, all to be used in the realm config smoke test. The output of this test is saved in the README.asciidoc. The result of this test is deterministic, re-running it produces the same realm config, client secret, and signed JWT.